lib/scripts/services/contextMenu.js

```diff
@@ -1,19 +1,57 @@
+function _checkIfWordCountNotOne(inputString){
+    var s = inputString;
+    s = s.replace(/(^\s*)|(\s*$)/gi,"");
+    s = s.replace(/[ ]{2,}/gi," ");
+    s = s.replace(/\n /,"\n");
+    if (s.split(' ').length != 1){
+        alert('Please select a single word only. Phrases such as: "' + inputString + '" aren\'t allowed');
+        return true;
+    }
+    return false;
+}
+
+function _currentURLIsBlacklistedOrMTWIsOff(documentURL) {
```

Comment:
I did spend some time on this with no success. 
I'll look into this again, but I am not being able to find a solution to dynamically enable/disable chrome extension options.